uint32[] missing_packets
